controllers/provider-aws/example/Infrastructure.yaml

```diff
@@ -1,4 +1,19 @@
 ---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: shoot--foo--bar
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: shoot--foo--bar
+  name: cloudprovider
+type: Opaque
+data:
+# accessKeyID: base64(access-key-id)
+# secretAccessKey: base64(secret-access-key)
+---
```

Comment:
Can we have `metadata.namespace=shoot--foo--bar` for `Cluster` and `Infrastructure` resource as well?